fix: duckdb cleanup (#54)
sourmash-bio/branchwater: metadata/README.md

> Building the duckdb database for the webapp to access is dependent on bigquery.

Reviewer: not dependent anymore?
```suggestion
Building the duckdb database for the webapp can be done via bigquery.
```
